planning/mission_planner/include/mission_planner/mission_planner_plugin.hpp

```diff
@@ -31,10 +32,13 @@ class PlannerPlugin
 public:
   using RoutePoints = std::vector<geometry_msgs::msg::Pose>;
   using HADMapRoute = autoware_auto_planning_msgs::msg::HADMapRoute;
+  using HADMapBin = autoware_auto_mapping_msgs::msg::HADMapBin;
   using MarkerArray = visualization_msgs::msg::MarkerArray;
 
   virtual ~PlannerPlugin() = default;
   virtual void initialize(rclcpp::Node * node) = 0;
+  virtual void initialize(
+    rclcpp::Node * node, const autoware_auto_mapping_msgs::msg::HADMapBin::ConstSharedPtr msg) = 0;
```

Comment:
Namespace can be removed because `using HADMapBin`.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/2259/commits/4a49ea8245b941cbfa8f6b48ac51dede0c155019
DONE

Reply:
It's same for the other two lines.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/2259/commits/764bc2ffc47ff099d4c1dcd0f8b800b2af88cae3
DONE